typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -26,11 +31,84 @@ import mantaTIAAddresses from './warp/manta-TIA-addresses.json';
 
 const repo = 'gcr.io/abacus-labs-dev/hyperlane-agent';
 
+// The chains here must be consistent with the environment's supportedChainNames, which is
+// checked / enforced at runtime & in the CI pipeline.
+//
+// This is intentionally separate and not derived from the environment's supportedChainNames
+// to allow for more fine-grained control over which chains are enabled for each agent role.
+export const hyperlaneContextAgentChainConfig: AgentChainConfig = {
+  [Role.Validator]: {
+    [Chains.arbitrum]: true,
+    [Chains.avalanche]: true,
+    [Chains.bsc]: true,
+    [Chains.celo]: true,
+    [Chains.ethereum]: true,
+    [Chains.neutron]: true,
+    [Chains.mantapacific]: true,
```

Comment:
shouldn't these be false too, to match the relayer?

Reply:
we generally run all production validators in the hyperlane context and leave the other contexts for just relaying, added a comment